docs/UserGuide.md

```diff
@@ -80,15 +80,15 @@ Adds a task to ModuleBook3.5.
 Format: `add n/TASK NAME m/MODULE d/DESCRIPTION b/DEADLINE [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-Enter the time you wish to complete your task in the format: YYYY-mm-dd HH:mm
+Enter the time you wish to complete your task in the format: yyyy-MM-dd HH:mm or yyyy-MM-dd (both accepted) 
```

Comment:
I think the UG should also mention what happens when the time is not keyed in. E.g. The current time on your computer is taken.

Reply:
ok will do

Reply:
done